### ISSUE TITLE
Complete AnimatedValueTest

### ISSUE THREAD

**Issue by AlmasB** [AlmasB/FXGL#948]
Currently the following code is not tested:

```
class AnimatedStringIncreasing(value: String) : AnimatedValue<String>("", value) {

    override fun animate(val1: String, val2: String, progress: Double, interpolator: Interpolator): String {
        var index = (val2.length * interpolator.interpolate(0.0, 1.0, progress)).toInt()

        index = max(index, 0)
        index = min(index, val2.length)

        return val2.substring(0, index)
    }
}

class AnimatedStringDecreasing(value: String) : AnimatedValue<String>("", value) {

    private val anim = AnimatedStringIncreasing(value)

    override fun animate(val1: String, val2: String, progress: Double, interpolator: Interpolator): String {
        return anim.animate(val1, val2, 1 - progress, interpolator)
    }
}
```

The tests should be added to AnimatedValueTest in fxgl-animation

**Comment by adambocco** [comment]
@AlmasB 
I'm new to open-source contribution and would love to help out with this.

**Comment by AlmasB** [comment]
Cool! Go for it!

Let me know if you get stuck

**Comment by AlmasB** [comment]
Closed by #951
